packages/suite/src/views/onboarding/steps/Final/index.tsx

```diff
@@ -158,6 +166,22 @@ export const FinalStep = () => {
         setState(null);
     };
 
+    const onChange = (value: string) => {
```

Comment:
It looks like the original change handler was typed incorrectly. I suggest typing it this way and just passing the function name to the `onChange` prop instead of inlining it.

```suggestion
    const handleChange: React.ChangeEventHandler<HTMLInputElement> = ({ target: { value } }) => {
```
Also, `handleChange` sounds like a better function name than `onChange`, but I'll leave that up to you.

Reply:
We maybe should introduce some code guidelines regarding this because I saw at least 3 different implementations of onChange functions

Reply:
Agreed! I don't know if you've seen our [code style guide](https://www.notion.so/satoshilabs/Code-style-guide-4cf109fe3c0d4d65a42dbd82ba4b0104). This is indeed not covered, feel free to add a proposal.